crates/test/src/rt/mod.rs

```diff
@@ -150,6 +151,12 @@ struct State {
     formatter: Box<dyn Formatter>,
 }
 
+enum Failure {
+    Error(JsValue),
+    ShouldPanic,
+    ShouldPanicExpected,
```

Comment:
It's not clear to a first-time reader what these variants are supposed to be. I can't come up with a better name so can you add a line of doc comment explaining what they mean?

Reply:
Done!